OneCommunityGlobal/HighestGoodNetworkApp
# Shengwei fix User Profile Quick Setup Modal

# Description
https://www.loom.com/share/bd2ded05bb324e7b8e1584e97d4d8f2f?sid=8e27bf62-3483-4f20-871a-5283a3101870

Fix QSC team code validation.  Team codes were originally retrieving from the Team data model. Now, we collect distinct team codes from all records within the UserProfile data model.


## Related PRS (if any):
This frontend PR is related to the #[1061 ](https://github.com/OneCommunityGlobal/HGNRest/pull/1061)backend PR.


## Main changes explained:
Fix QSC team code validation.  Team codes were originally retrieving from the Team data model. Now, we collect distinct team codes from all records within the UserProfile data model.

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin user
5. go to user profile and test the QSC modal by clicking Add a New Title Button.
<img width="664" alt="image" src="https://github.com/user-attachments/assets/2bebac74-2da1-4404-ab1c-e9600f012d37">


## Screenshots or videos of changes:
Demo

https://github.com/user-attachments/assets/904ce46f-865d-4b5c-b03d-546fbd4dc633



## Note:
Include the information the reviewers need to know.
